meta/group-info/id.yaml

```diff
@@ -1,11 +1,63 @@
-outdated: true # missing entries
 separator: ", "
+alumni:
+  roles:
+    academy: Pendiri osu!academy
+    admin: Administrator
+    chat: Moderator Chat
```

Comment:
I think you might want to do "Chat Moderator" here, since you preferred to keep the English titles in other PR, and this is  referring to the user title people had